src/molecules/dialog/index.tsx

```diff
@@ -82,3 +87,30 @@ export default function AutoformDialog({
     </Dialog>
   );
 }
+
+export type SubContentDialogProps = {
+  action: TableActionCommon & TableActionSubContentDialog;
+  onOpenChange: (e: boolean) => void;
+  open: boolean;
+};
+export const SubContentDialog = ({
+  open,
+  onOpenChange,
+  action,
+}: SubContentDialogProps) => (
+  <Dialog open={open} onOpenChange={onOpenChange}>
+    <DialogContent className="sm:max-w-[425px] max-h-[90vh] overflow-auto [&::-webkit-scrollbar]:hidden">
+      <DialogHeader>
+        <DialogTitle>{action?.cta}</DialogTitle>
+        <DialogDescription>{action?.description}</DialogDescription>
+      </DialogHeader>
+      <div className="grid gap-4 py-4">
+        {action.content ? (
+          action.content
+        ) : (
+          <Spinner fullScreen={false} variant="transparent" />
+        )}
+      </div>
+    </DialogContent>
+  </Dialog>
+);
```

Comment:
The logic we have here is similar to the atuforomDialog logic, can we merge both of them?